ProjectSettings/EditorBuildSettings.asset

```diff
@@ -18,18 +18,27 @@ EditorBuildSettings:
     path: Assets/MunroHoberman/WerewolfReviversAssets/Scenes/WerewolfReviversTitle.unity
     guid: 2cc38a3472aff87488e28d38e6af8388
   - enabled: 0
-    path: Assets/AishaBikebayeva/Scenes/WerewolfReviversGame.unity
+    path: 
     guid: e05ba7fcb90e447469f1f5c91d18d7db
   - enabled: 0
-    path: Assets/AishaBikebayeva/Scenes/WerewolfReviversTitle.unity
+    path: 
     guid: e8bf3e053ed064965b17683b34dc55b1
   - enabled: 0
-    path: Assets/AishaBikebayeva/Scenes/WerewolfReviversLose.unity
+    path: 
     guid: 47ac0cb2a332846768133bda92b1cbb0
   - enabled: 0
-    path: Assets/AishaBikebayeva/Scenes/WerewolfReviversWin.unity
+    path: 
     guid: e46eadf6808744836bf2147d7a4af55c
   - enabled: 1
     path: Assets/JasonLi/ARCADE/BasicScene.unity
     guid: a9f602ea32ef9ce469489c9b7cf9da64
+  - enabled: 1
```

Comment:
This is fine to have but it will cause merge conflicts later, so you'll have to update these if someone merges before you